gatsby-browser.js

```diff
@@ -13,6 +13,8 @@ import TestProvider from './src/sdk/tests'
 import { uiActions, uiEffects, uiInitialState } from './src/sdk/ui'
 import storeConfig from './store.config'
 
+const channel = JSON.stringify(storeConfig.channel)
```

Comment:
stringify on store.config.js so we dont need to do this on both gatsby-ssr and gatsby-browser

Reply:
Fixed here, but parse on server. Ok!?